Gemfile

```diff
@@ -240,7 +240,6 @@ gem 'aws-sdk-glue'
 gem 'aws-sdk-rds'
 gem 'aws-sdk-route53'
 gem 'aws-sdk-s3'
-gem 'aws-sdk-sagemaker'
```

Comment:
Can you also run `bundle install` locally and include any `Gemfile.lock` changes?